lib/daru/dataframe.rb

```diff
@@ -1419,6 +1422,29 @@ def report_building(b) # :nodoc: #
         end
       end
     end
+    #custom function for dataframe summary
+    def g_summary()
+       frame_summary = "= \t\n"
```

Comment:
`df_summary` is better. Why the `\t\n` at the end? Have you seen how exactly ReportBuilder prepares reports?